basics/Main.java

```diff
@@ -0,0 +1,81 @@
+import java.time.LocalDateTime;
+
+public class Main {
+
+
+public static void main(String[] args){
+int dogCount = 1;
+int catCount = 2;
+int turtleCount = 0;
+
+
+System.out.println("I own " + dogCount + " " + pluralize(dogCount, "dog") + ".");
+
+System.out.println("I own " + catCount + " " + pluralize(catCount,"cat") + ".");
+
+
+System.out.println("I own " + turtleCount + " " + pluralize(turtleCount, "turtle" ) + ".");
+
+
+flipNHeads(1);
+
+Main object = new Main();
+object.clock();
+
+}
+
+public static String pluralize(int number, String word) {
+    
+if (number==1) {
+return word;
+} else {
+return word + "s";
+}
+}
+
+
+public static void flipNHeads(int n) {
+    
+int counter=0;
+int headCounter = 0;
+
+while (headCounter<n) {
+    counter ++;
+    double rand = Math.random();
+    if (rand<0.5) {
+        System.out.println("tails");
+    } else {
+        System.out.println("Heads");
+        headCounter ++;
+    }
+
+}
+
+System.out.println("It took " + counter+ " flips to flip " + n + " head in a row.");
```

Comment:
Oh no! The code you have now is counting it as N heads "in a row" even if they weren't in a row, e.g. it's counting as 3 heads in a row even if the pattern was HTHTH. You'll probably need to think about modifying your headCounter when you get tails to solve this problem.